core_lib/src/tool/basetool.h

```diff
@@ -50,6 +50,7 @@ class Properties
     int   stabilizerLevel = 0;
     qreal tolerance = 0;
     bool  useFillContour = false;
+    bool  showInfo = true;
```

Comment:
This name is very ambiguous, please make it describe what the property does.

Reply:
I've changed it to showSelectionInfo, which I call it in settings as well. Should be understandable.

Reply:
That's fine 👍